2017-11-09 11:01:00.730 [info] <0.560.0> Application lager started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:01:00.730 [info] <0.560.0> Application ranch started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:01:00.730 [info] <0.560.0> Application ranch_proxy_protocol started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:01:00.730 [info] <0.560.0> Application cowboy started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:01:00.730 [info] <0.560.0> Application rabbit_common started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:01:00.730 [info] <0.560.0> Application amqp_client started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:01:00.730 [info] <0.560.0> Application pampa_pro started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:01:00.732 [info] <0.560.0> Application sasl started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:01:00.733 [info] <0.560.0> Application runtime_tools started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:03:43.764 [error] <0.616.0> /home/pampa/pampa_pro/src/pampa_pro_app.erl:9: Error: record amqp_params_direct undefined
/home/pampa/pampa_pro/src/pampa_pro_app.erl:9: Warning: variable 'Connection' is unused
2017-11-09 11:04:14.138 [warning] <0.616.0> /home/pampa/pampa_pro/src/pampa_pro_app.erl:10: Warning: variable 'Connection' is unused
/home/pampa/pampa_pro/src/pampa_pro_app.erl:0: Recompiled with 1 warnings
2017-11-09 11:04:14.154 [info] <0.616.0> pampa_pro_app: Reloaded! (Beam changed.)
2017-11-09 11:04:53.624 [info] <0.616.0> /home/pampa/pampa_pro/src/pampa_pro_app.erl:0: Recompiled.
2017-11-09 11:04:53.642 [info] <0.616.0> pampa_pro_app: Reloaded! (Beam changed.)
2017-11-09 11:05:21.950 [info] <0.616.0> /home/pampa/pampa_pro/src/pampa_pro_app.erl:0: Recompiled.
2017-11-09 11:05:55.313 [info] <0.616.0> /home/pampa/pampa_pro/src/pampa_pro_app.erl:0: Recompiled.
2017-11-09 11:05:55.326 [info] <0.616.0> pampa_pro_app: Reloaded! (Beam changed.)
2017-11-09 11:06:15.561 [info] <0.616.0> /home/pampa/pampa_pro/src/pampa_pro_app.erl:0: Recompiled.
2017-11-09 11:06:28.792 [info] <0.560.0> Application lager started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:06:28.793 [info] <0.560.0> Application ranch started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:06:28.793 [info] <0.560.0> Application ranch_proxy_protocol started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:06:28.793 [info] <0.560.0> Application cowboy started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:06:28.793 [info] <0.560.0> Application rabbit_common started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:06:28.793 [info] <0.560.0> Application amqp_client started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:06:28.794 [error] <0.673.0>@amqp_gen_connection:terminate:239 gen_server <0.673.0> terminated with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,'pampa_pro@127.0.0.1',none,[]}}) line 239
2017-11-09 11:06:28.794 [error] <0.673.0>@amqp_gen_connection:terminate:239 CRASH REPORT Process <0.673.0> with 0 neighbours crashed with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,'pampa_pro@127.0.0.1',none,[]}}) line 239
2017-11-09 11:06:28.795 [error] <0.671.0>@amqp_gen_connection:terminate:239 Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,'pampa_pro@127.0.0.1',none,[]}) at <0.673.0> exit with reason no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,'pampa_pro@127.0.0.1',none,[]}}) line 239 in context child_terminated
2017-11-09 11:06:28.795 [error] <0.671.0> Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,'pampa_pro@127.0.0.1',none,[]}) at <0.673.0> exit with reason reached_max_restart_intensity in context shutdown
2017-11-09 11:06:28.795 [error] <0.669.0> CRASH REPORT Process <0.669.0> with 0 neighbours exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@127.0.0.1.2.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510196788794},{node,'pampa_pro@127.0.0.1'},{user_who_performed_action,none}]],[]},{...},...]}}}},...},...],...},...]},...} in application_master:init/4 line 134
2017-11-09 11:06:28.795 [info] <0.560.0> Application pampa_pro exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@127.0.0.1.2.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510196788794},{node,'pampa_pro@127.0.0.1'},{user_who_performed_action,none}]],[]},{...},...]}}}},...},...],...},...]},...}
2017-11-09 11:07:28.418 [info] <0.560.0> Application lager started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:07:28.418 [info] <0.560.0> Application ranch started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:07:28.419 [info] <0.560.0> Application ranch_proxy_protocol started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:07:28.419 [info] <0.560.0> Application cowboy started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:07:28.419 [info] <0.560.0> Application rabbit_common started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:07:28.419 [info] <0.560.0> Application amqp_client started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:07:28.435 [info] <0.560.0> Application pampa_pro started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:07:28.437 [info] <0.560.0> Application sasl started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:07:28.438 [info] <0.560.0> Application runtime_tools started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:07:56.481 [info] <0.616.0> /home/pampa/pampa_pro/src/pampa_pro_app.erl:0: Recompiled.
2017-11-09 11:07:56.493 [info] <0.616.0> pampa_pro_app: Reloaded! (Beam changed.)
2017-11-09 11:08:09.503 [info] <0.560.0> Application lager started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:08:09.503 [info] <0.560.0> Application ranch started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:08:09.503 [info] <0.560.0> Application ranch_proxy_protocol started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:08:09.504 [info] <0.560.0> Application cowboy started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:08:09.504 [info] <0.560.0> Application rabbit_common started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:08:09.504 [info] <0.560.0> Application amqp_client started on node 'pampa_pro@127.0.0.1'
2017-11-09 11:08:09.505 [error] <0.673.0>@amqp_gen_connection:terminate:239 gen_server <0.673.0> terminated with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,'pampa_pro@127.0.0.1',none,[]}}) line 239
2017-11-09 11:08:09.505 [error] <0.673.0>@amqp_gen_connection:terminate:239 CRASH REPORT Process <0.673.0> with 0 neighbours crashed with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,'pampa_pro@127.0.0.1',none,[]}}) line 239
2017-11-09 11:08:09.505 [error] <0.671.0>@amqp_gen_connection:terminate:239 Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,'pampa_pro@127.0.0.1',none,[]}) at <0.673.0> exit with reason no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,'pampa_pro@127.0.0.1',none,[]}}) line 239 in context child_terminated
2017-11-09 11:08:09.506 [error] <0.671.0> Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,'pampa_pro@127.0.0.1',none,[]}) at <0.673.0> exit with reason reached_max_restart_intensity in context shutdown
2017-11-09 11:08:09.506 [error] <0.669.0> CRASH REPORT Process <0.669.0> with 0 neighbours exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@127.0.0.1.1.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510196889505},{node,'pampa_pro@127.0.0.1'},{user_who_performed_action,none}]],[]},{...},...]}}}},...},...],...},...]},...} in application_master:init/4 line 134
2017-11-09 11:08:09.506 [info] <0.560.0> Application pampa_pro exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@127.0.0.1.1.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510196889505},{node,'pampa_pro@127.0.0.1'},{user_who_performed_action,none}]],[]},{...},...]}}}},...},...],...},...]},...}
2017-11-09 11:08:59.868 [info] <0.560.0> Application lager started on node pampa_pro@localhost
2017-11-09 11:08:59.868 [info] <0.560.0> Application ranch started on node pampa_pro@localhost
2017-11-09 11:08:59.868 [info] <0.560.0> Application ranch_proxy_protocol started on node pampa_pro@localhost
2017-11-09 11:08:59.869 [info] <0.560.0> Application cowboy started on node pampa_pro@localhost
2017-11-09 11:08:59.869 [info] <0.560.0> Application rabbit_common started on node pampa_pro@localhost
2017-11-09 11:08:59.869 [info] <0.560.0> Application amqp_client started on node pampa_pro@localhost
2017-11-09 11:08:59.869 [error] <0.673.0>@amqp_gen_connection:terminate:239 gen_server <0.673.0> terminated with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}}) line 239
2017-11-09 11:08:59.870 [error] <0.673.0>@amqp_gen_connection:terminate:239 CRASH REPORT Process <0.673.0> with 0 neighbours crashed with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}}) line 239
2017-11-09 11:08:59.870 [error] <0.671.0>@amqp_gen_connection:terminate:239 Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}) at <0.673.0> exit with reason no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}}) line 239 in context child_terminated
2017-11-09 11:08:59.870 [error] <0.671.0> Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}) at <0.673.0> exit with reason reached_max_restart_intensity in context shutdown
2017-11-09 11:08:59.870 [error] <0.669.0> CRASH REPORT Process <0.669.0> with 0 neighbours exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@localhost.2.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510196939869},{node,pampa_pro@localhost},{user_who_performed_action,none}]],[]},{rpc,...},...]}}}},...},...],...},...]},...} in application_master:init/4 line 134
2017-11-09 11:08:59.870 [info] <0.560.0> Application pampa_pro exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@localhost.2.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510196939869},{node,pampa_pro@localhost},{user_who_performed_action,none}]],[]},{rpc,...},...]}}}},...},...],...},...]},...}
2017-11-09 11:11:25.877 [info] <0.560.0> Application lager started on node pampa_pro@VM_0_14_centos
2017-11-09 11:11:25.878 [info] <0.560.0> Application ranch started on node pampa_pro@VM_0_14_centos
2017-11-09 11:11:25.878 [info] <0.560.0> Application ranch_proxy_protocol started on node pampa_pro@VM_0_14_centos
2017-11-09 11:11:25.878 [info] <0.560.0> Application cowboy started on node pampa_pro@VM_0_14_centos
2017-11-09 11:11:25.878 [info] <0.560.0> Application rabbit_common started on node pampa_pro@VM_0_14_centos
2017-11-09 11:11:25.878 [info] <0.560.0> Application amqp_client started on node pampa_pro@VM_0_14_centos
2017-11-09 11:11:25.878 [error] <0.673.0>@amqp_gen_connection:terminate:239 gen_server <0.673.0> terminated with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@VM_0_14_centos,none,[]}}) line 239
2017-11-09 11:11:25.879 [error] <0.673.0>@amqp_gen_connection:terminate:239 CRASH REPORT Process <0.673.0> with 0 neighbours crashed with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@VM_0_14_centos,none,[]}}) line 239
2017-11-09 11:11:25.879 [error] <0.671.0>@amqp_gen_connection:terminate:239 Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,pampa_pro@VM_0_14_centos,none,[]}) at <0.673.0> exit with reason no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@VM_0_14_centos,none,[]}}) line 239 in context child_terminated
2017-11-09 11:11:25.879 [error] <0.671.0> Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,pampa_pro@VM_0_14_centos,none,[]}) at <0.673.0> exit with reason reached_max_restart_intensity in context shutdown
2017-11-09 11:11:25.879 [error] <0.669.0> CRASH REPORT Process <0.669.0> with 0 neighbours exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@VM_0_14_centos.3.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510197085878},{node,pampa_pro@VM_0_14_centos},{user_who_performed_action,none}]],...},...]}}}},...},...],...},...]},...} in application_master:init/4 line 134
2017-11-09 11:11:25.880 [info] <0.560.0> Application pampa_pro exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@VM_0_14_centos.3.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510197085878},{node,pampa_pro@VM_0_14_centos},{user_who_performed_action,none}]],...},...]}}}},...},...],...},...]},...}
2017-11-09 11:13:25.720 [info] <0.560.0> Application lager started on node pampa_pro@VM_0_14_centos
2017-11-09 11:13:25.720 [info] <0.560.0> Application ranch started on node pampa_pro@VM_0_14_centos
2017-11-09 11:13:25.720 [info] <0.560.0> Application ranch_proxy_protocol started on node pampa_pro@VM_0_14_centos
2017-11-09 11:13:25.720 [info] <0.560.0> Application cowboy started on node pampa_pro@VM_0_14_centos
2017-11-09 11:13:25.720 [info] <0.560.0> Application rabbit_common started on node pampa_pro@VM_0_14_centos
2017-11-09 11:13:25.721 [info] <0.560.0> Application amqp_client started on node pampa_pro@VM_0_14_centos
2017-11-09 11:13:25.721 [error] <0.673.0>@amqp_gen_connection:terminate:239 gen_server <0.673.0> terminated with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@VM_0_14_centos,none,[]}}) line 239
2017-11-09 11:13:25.721 [error] <0.673.0>@amqp_gen_connection:terminate:239 CRASH REPORT Process <0.673.0> with 0 neighbours crashed with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@VM_0_14_centos,none,[]}}) line 239
2017-11-09 11:13:25.721 [error] <0.671.0>@amqp_gen_connection:terminate:239 Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,pampa_pro@VM_0_14_centos,none,[]}) at <0.673.0> exit with reason no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@VM_0_14_centos,none,[]}}) line 239 in context child_terminated
2017-11-09 11:13:25.721 [error] <0.671.0> Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,pampa_pro@VM_0_14_centos,none,[]}) at <0.673.0> exit with reason reached_max_restart_intensity in context shutdown
2017-11-09 11:13:25.722 [error] <0.669.0> CRASH REPORT Process <0.669.0> with 0 neighbours exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@VM_0_14_centos.1.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510197205720},{node,pampa_pro@VM_0_14_centos},{user_who_performed_action,none}]],...},...]}}}},...},...],...},...]},...} in application_master:init/4 line 134
2017-11-09 11:13:25.722 [info] <0.560.0> Application pampa_pro exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@VM_0_14_centos.1.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510197205720},{node,pampa_pro@VM_0_14_centos},{user_who_performed_action,none}]],...},...]}}}},...},...],...},...]},...}
2017-11-09 11:16:16.748 [info] <0.560.0> Application lager started on node pampa_pro@localhost
2017-11-09 11:16:16.748 [info] <0.560.0> Application ranch started on node pampa_pro@localhost
2017-11-09 11:16:16.748 [info] <0.560.0> Application ranch_proxy_protocol started on node pampa_pro@localhost
2017-11-09 11:16:16.748 [info] <0.560.0> Application cowboy started on node pampa_pro@localhost
2017-11-09 11:16:16.749 [info] <0.560.0> Application rabbit_common started on node pampa_pro@localhost
2017-11-09 11:16:16.749 [info] <0.560.0> Application amqp_client started on node pampa_pro@localhost
2017-11-09 11:16:16.750 [error] <0.673.0>@amqp_gen_connection:terminate:239 gen_server <0.673.0> terminated with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}}) line 239
2017-11-09 11:16:16.750 [error] <0.673.0>@amqp_gen_connection:terminate:239 CRASH REPORT Process <0.673.0> with 0 neighbours crashed with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}}) line 239
2017-11-09 11:16:16.751 [error] <0.671.0>@amqp_gen_connection:terminate:239 Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}) at <0.673.0> exit with reason no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}}) line 239 in context child_terminated
2017-11-09 11:16:16.751 [error] <0.671.0> Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}) at <0.673.0> exit with reason reached_max_restart_intensity in context shutdown
2017-11-09 11:16:16.751 [error] <0.669.0> CRASH REPORT Process <0.669.0> with 0 neighbours exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@localhost.2.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510197376749},{node,pampa_pro@localhost},{user_who_performed_action,none}]],[]},{rpc,...},...]}}}},...},...],...},...]},...} in application_master:init/4 line 134
2017-11-09 11:16:16.751 [info] <0.560.0> Application pampa_pro exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@localhost.2.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510197376749},{node,pampa_pro@localhost},{user_who_performed_action,none}]],[]},{rpc,...},...]}}}},...},...],...},...]},...}
2017-11-09 11:37:42.114 [info] <0.560.0> Application lager started on node pampa_pro@localhost
2017-11-09 11:37:42.114 [info] <0.560.0> Application ranch started on node pampa_pro@localhost
2017-11-09 11:37:42.114 [info] <0.560.0> Application ranch_proxy_protocol started on node pampa_pro@localhost
2017-11-09 11:37:42.114 [info] <0.560.0> Application cowboy started on node pampa_pro@localhost
2017-11-09 11:37:42.114 [info] <0.560.0> Application rabbit_common started on node pampa_pro@localhost
2017-11-09 11:37:42.114 [info] <0.560.0> Application amqp_client started on node pampa_pro@localhost
2017-11-09 11:37:42.115 [error] <0.673.0>@amqp_gen_connection:terminate:239 gen_server <0.673.0> terminated with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}}) line 239
2017-11-09 11:37:42.115 [error] <0.673.0>@amqp_gen_connection:terminate:239 CRASH REPORT Process <0.673.0> with 0 neighbours crashed with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}}) line 239
2017-11-09 11:37:42.116 [error] <0.671.0>@amqp_gen_connection:terminate:239 Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}) at <0.673.0> exit with reason no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}}) line 239 in context child_terminated
2017-11-09 11:37:42.116 [error] <0.671.0> Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}) at <0.673.0> exit with reason reached_max_restart_intensity in context shutdown
2017-11-09 11:37:42.116 [error] <0.669.0> CRASH REPORT Process <0.669.0> with 0 neighbours exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@localhost.3.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510198662115},{node,pampa_pro@localhost},{user_who_performed_action,none}]],[]},{rpc,...},...]}}}},...},...],...},...]},...} in application_master:init/4 line 134
2017-11-09 11:37:42.117 [info] <0.560.0> Application pampa_pro exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@localhost.3.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510198662115},{node,pampa_pro@localhost},{user_who_performed_action,none}]],[]},{rpc,...},...]}}}},...},...],...},...]},...}
2017-11-09 11:38:59.503 [info] <0.560.0> Application lager started on node pampa_pro@localhost
2017-11-09 11:38:59.503 [info] <0.560.0> Application ranch started on node pampa_pro@localhost
2017-11-09 11:38:59.503 [info] <0.560.0> Application ranch_proxy_protocol started on node pampa_pro@localhost
2017-11-09 11:38:59.503 [info] <0.560.0> Application cowboy started on node pampa_pro@localhost
2017-11-09 11:38:59.503 [info] <0.560.0> Application rabbit_common started on node pampa_pro@localhost
2017-11-09 11:38:59.503 [info] <0.560.0> Application amqp_client started on node pampa_pro@localhost
2017-11-09 11:38:59.503 [error] <0.669.0>@amqp_connection:start_direct CRASH REPORT Process <0.669.0> with 0 neighbours exited with reason: call to undefined function amqp_connection:start_direct() in application_master:init/4 line 134
2017-11-09 11:38:59.504 [info] <0.560.0>@amqp_connection:start_direct Application pampa_pro exited with reason: call to undefined function amqp_connection:start_direct()
2017-11-09 11:45:43.225 [info] <0.560.0> Application lager started on node pampa_pro@localhost
2017-11-09 11:45:43.225 [info] <0.560.0> Application ranch started on node pampa_pro@localhost
2017-11-09 11:45:43.225 [info] <0.560.0> Application ranch_proxy_protocol started on node pampa_pro@localhost
2017-11-09 11:45:43.225 [info] <0.560.0> Application cowboy started on node pampa_pro@localhost
2017-11-09 11:45:43.225 [info] <0.560.0> Application rabbit_common started on node pampa_pro@localhost
2017-11-09 11:45:43.225 [info] <0.560.0> Application amqp_client started on node pampa_pro@localhost
2017-11-09 11:45:43.226 [error] <0.673.0>@amqp_gen_connection:terminate:239 gen_server <0.673.0> terminated with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}}) line 239
2017-11-09 11:45:43.226 [error] <0.673.0>@amqp_gen_connection:terminate:239 CRASH REPORT Process <0.673.0> with 0 neighbours crashed with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}}) line 239
2017-11-09 11:45:43.226 [error] <0.671.0>@amqp_gen_connection:terminate:239 Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}) at <0.673.0> exit with reason no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}}) line 239 in context child_terminated
2017-11-09 11:45:43.226 [error] <0.671.0> Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}) at <0.673.0> exit with reason reached_max_restart_intensity in context shutdown
2017-11-09 11:45:43.226 [error] <0.669.0> CRASH REPORT Process <0.669.0> with 0 neighbours exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@localhost.2.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510199143225},{node,pampa_pro@localhost},{user_who_performed_action,none}]],[]},{rpc,...},...]}}}},...},...],...},...]},...} in application_master:init/4 line 134
2017-11-09 11:45:43.227 [info] <0.560.0> Application pampa_pro exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@localhost.2.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510199143225},{node,pampa_pro@localhost},{user_who_performed_action,none}]],[]},{rpc,...},...]}}}},...},...],...},...]},...}
2017-11-09 11:54:17.972 [info] <0.560.0> Application lager started on node pampa_pro@localhost
2017-11-09 11:54:17.973 [info] <0.560.0> Application ranch started on node pampa_pro@localhost
2017-11-09 11:54:17.973 [info] <0.560.0> Application ranch_proxy_protocol started on node pampa_pro@localhost
2017-11-09 11:54:17.973 [info] <0.560.0> Application cowboy started on node pampa_pro@localhost
2017-11-09 11:54:17.973 [info] <0.560.0> Application rabbit_common started on node pampa_pro@localhost
2017-11-09 11:54:17.973 [info] <0.560.0> Application amqp_client started on node pampa_pro@localhost
2017-11-09 11:54:17.973 [error] <0.673.0>@amqp_gen_connection:terminate:239 gen_server <0.673.0> terminated with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{{function_clause,[{gen,do_for_proc,[{rex,"localhost"},#Fun<gen.0.9..>],...},...]},...}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,"/","localhost",none,[]}}) line 239
2017-11-09 11:54:17.973 [error] <0.673.0>@amqp_gen_connection:terminate:239 CRASH REPORT Process <0.673.0> with 0 neighbours crashed with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{{function_clause,[{gen,do_for_proc,[{rex,"localhost"},#Fun<gen.0.9..>],...},...]},...}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,"/","localhost",none,[]}}) line 239
2017-11-09 11:54:17.974 [error] <0.671.0>@amqp_gen_connection:terminate:239 Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,"/","localhost",none,[]}) at <0.673.0> exit with reason no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{{function_clause,[{gen,do_for_proc,[{rex,"localhost"},#Fun<gen.0.9..>],...},...]},...}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,"/","localhost",none,[]}}) line 239 in context child_terminated
2017-11-09 11:54:17.974 [error] <0.671.0> Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,"/","localhost",none,[]}) at <0.673.0> exit with reason reached_max_restart_intensity in context shutdown
2017-11-09 11:54:17.974 [error] <0.669.0> CRASH REPORT Process <0.669.0> with 0 neighbours exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{{function_clause,[{gen,do_for_proc,[{rex,"localhost"},#Fun<gen.0.94381411>],[{file,"gen.erl"},{line,253}]},{gen_server,call,3,[{file,"gen_server.erl"},{line,210}]},{rpc,do_call,3,[{file,"rpc.erl"},{line,327}]},{amqp_direct_connection,connect,4,[{file,"src/amqp_direct_connection.erl"},{line,151}]},{amqp_gen_connection,handle_call,3,[{file,"src/amqp_gen_connection.erl"},{line,171}]},{gen_server,try_handle_call,...},...]},...}}}},...},...],...},...]},...} in application_master:init/4 line 134
2017-11-09 11:54:17.974 [info] <0.560.0> Application pampa_pro exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{{function_clause,[{gen,do_for_proc,[{rex,"localhost"},#Fun<gen.0.94381411>],[{file,"gen.erl"},{line,253}]},{gen_server,call,3,[{file,"gen_server.erl"},{line,210}]},{rpc,do_call,3,[{file,"rpc.erl"},{line,327}]},{amqp_direct_connection,connect,4,[{file,"src/amqp_direct_connection.erl"},{line,151}]},{amqp_gen_connection,handle_call,3,[{file,"src/amqp_gen_connection.erl"},{line,171}]},{gen_server,try_handle_call,...},...]},...}}}},...},...],...},...]},...}
2017-11-09 12:12:54.064 [info] <0.560.0> Application lager started on node pampa_pro@localhost
2017-11-09 12:12:54.064 [info] <0.560.0> Application ranch started on node pampa_pro@localhost
2017-11-09 12:12:54.064 [info] <0.560.0> Application ranch_proxy_protocol started on node pampa_pro@localhost
2017-11-09 12:12:54.065 [info] <0.560.0> Application cowboy started on node pampa_pro@localhost
2017-11-09 12:12:54.065 [info] <0.560.0> Application rabbit_common started on node pampa_pro@localhost
2017-11-09 12:12:54.065 [info] <0.560.0> Application amqp_client started on node pampa_pro@localhost
2017-11-09 12:12:54.065 [error] <0.673.0>@amqp_gen_connection:terminate:239 gen_server <0.673.0> terminated with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{{function_clause,[{gen,do_for_proc,[{rex,"localhost"},#Fun<gen.0.9..>],...},...]},...}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,"/","localhost",none,[]}}) line 239
2017-11-09 12:12:54.066 [error] <0.673.0>@amqp_gen_connection:terminate:239 CRASH REPORT Process <0.673.0> with 0 neighbours crashed with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{{function_clause,[{gen,do_for_proc,[{rex,"localhost"},#Fun<gen.0.9..>],...},...]},...}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,"/","localhost",none,[]}}) line 239
2017-11-09 12:12:54.066 [error] <0.671.0>@amqp_gen_connection:terminate:239 Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,"/","localhost",none,[]}) at <0.673.0> exit with reason no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{{function_clause,[{gen,do_for_proc,[{rex,"localhost"},#Fun<gen.0.9..>],...},...]},...}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,"/","localhost",none,[]}}) line 239 in context child_terminated
2017-11-09 12:12:54.066 [error] <0.671.0> Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,"/","localhost",none,[]}) at <0.673.0> exit with reason reached_max_restart_intensity in context shutdown
2017-11-09 12:12:54.066 [error] <0.669.0> CRASH REPORT Process <0.669.0> with 0 neighbours exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{{function_clause,[{gen,do_for_proc,[{rex,"localhost"},#Fun<gen.0.94381411>],[{file,"gen.erl"},{line,253}]},{gen_server,call,3,[{file,"gen_server.erl"},{line,210}]},{rpc,do_call,3,[{file,"rpc.erl"},{line,327}]},{amqp_direct_connection,connect,4,[{file,"src/amqp_direct_connection.erl"},{line,151}]},{amqp_gen_connection,handle_call,3,[{file,"src/amqp_gen_connection.erl"},{line,171}]},{gen_server,try_handle_call,...},...]},...}}}},...},...],...},...]},...} in application_master:init/4 line 134
2017-11-09 12:12:54.066 [info] <0.560.0> Application pampa_pro exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{{function_clause,[{gen,do_for_proc,[{rex,"localhost"},#Fun<gen.0.94381411>],[{file,"gen.erl"},{line,253}]},{gen_server,call,3,[{file,"gen_server.erl"},{line,210}]},{rpc,do_call,3,[{file,"rpc.erl"},{line,327}]},{amqp_direct_connection,connect,4,[{file,"src/amqp_direct_connection.erl"},{line,151}]},{amqp_gen_connection,handle_call,3,[{file,"src/amqp_gen_connection.erl"},{line,171}]},{gen_server,try_handle_call,...},...]},...}}}},...},...],...},...]},...}
2017-11-09 12:16:03.278 [info] <0.560.0> Application lager started on node pampa_pro@localhost
2017-11-09 12:16:03.278 [info] <0.560.0> Application ranch started on node pampa_pro@localhost
2017-11-09 12:16:03.278 [info] <0.560.0> Application ranch_proxy_protocol started on node pampa_pro@localhost
2017-11-09 12:16:03.279 [info] <0.560.0> Application cowboy started on node pampa_pro@localhost
2017-11-09 12:16:03.279 [info] <0.560.0> Application rabbit_common started on node pampa_pro@localhost
2017-11-09 12:16:03.279 [info] <0.560.0> Application amqp_client started on node pampa_pro@localhost
2017-11-09 12:16:03.279 [error] <0.673.0>@amqp_gen_connection:terminate:239 gen_server <0.673.0> terminated with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{{function_clause,[{gen,do_for_proc,[{rex,"localhost"},#Fun<gen.0.9..>],...},...]},...}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,"/","localhost",none,[]}}) line 239
2017-11-09 12:16:03.280 [error] <0.673.0>@amqp_gen_connection:terminate:239 CRASH REPORT Process <0.673.0> with 0 neighbours crashed with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{{function_clause,[{gen,do_for_proc,[{rex,"localhost"},#Fun<gen.0.9..>],...},...]},...}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,"/","localhost",none,[]}}) line 239
2017-11-09 12:16:03.280 [error] <0.671.0>@amqp_gen_connection:terminate:239 Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,"/","localhost",none,[]}) at <0.673.0> exit with reason no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{{function_clause,[{gen,do_for_proc,[{rex,"localhost"},#Fun<gen.0.9..>],...},...]},...}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,"/","localhost",none,[]}}) line 239 in context child_terminated
2017-11-09 12:16:03.280 [error] <0.671.0> Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,"/","localhost",none,[]}) at <0.673.0> exit with reason reached_max_restart_intensity in context shutdown
2017-11-09 12:16:03.280 [error] <0.669.0> CRASH REPORT Process <0.669.0> with 0 neighbours exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{{function_clause,[{gen,do_for_proc,[{rex,"localhost"},#Fun<gen.0.94381411>],[{file,"gen.erl"},{line,253}]},{gen_server,call,3,[{file,"gen_server.erl"},{line,210}]},{rpc,do_call,3,[{file,"rpc.erl"},{line,327}]},{amqp_direct_connection,connect,4,[{file,"src/amqp_direct_connection.erl"},{line,151}]},{amqp_gen_connection,handle_call,3,[{file,"src/amqp_gen_connection.erl"},{line,171}]},{gen_server,try_handle_call,...},...]},...}}}},...},...],...},...]},...} in application_master:init/4 line 134
2017-11-09 12:16:03.281 [info] <0.560.0> Application pampa_pro exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{{function_clause,[{gen,do_for_proc,[{rex,"localhost"},#Fun<gen.0.94381411>],[{file,"gen.erl"},{line,253}]},{gen_server,call,3,[{file,"gen_server.erl"},{line,210}]},{rpc,do_call,3,[{file,"rpc.erl"},{line,327}]},{amqp_direct_connection,connect,4,[{file,"src/amqp_direct_connection.erl"},{line,151}]},{amqp_gen_connection,handle_call,3,[{file,"src/amqp_gen_connection.erl"},{line,171}]},{gen_server,try_handle_call,...},...]},...}}}},...},...],...},...]},...}
2017-11-09 12:21:27.293 [info] <0.560.0> Application lager started on node pampa_pro@localhost
2017-11-09 12:21:27.293 [info] <0.560.0> Application ranch started on node pampa_pro@localhost
2017-11-09 12:21:27.293 [info] <0.560.0> Application ranch_proxy_protocol started on node pampa_pro@localhost
2017-11-09 12:21:27.293 [info] <0.560.0> Application cowboy started on node pampa_pro@localhost
2017-11-09 12:21:27.293 [info] <0.560.0> Application rabbit_common started on node pampa_pro@localhost
2017-11-09 12:21:27.293 [info] <0.560.0> Application amqp_client started on node pampa_pro@localhost
2017-11-09 12:21:27.294 [error] <0.673.0>@amqp_gen_connection:terminate:239 gen_server <0.673.0> terminated with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{{function_clause,[{gen,do_for_proc,[{rex,"localhost"},#Fun<gen.0.9..>],...},...]},...}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,"/","localhost",none,[]}}) line 239
2017-11-09 12:21:27.294 [error] <0.673.0>@amqp_gen_connection:terminate:239 CRASH REPORT Process <0.673.0> with 0 neighbours crashed with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{{function_clause,[{gen,do_for_proc,[{rex,"localhost"},#Fun<gen.0.9..>],...},...]},...}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,"/","localhost",none,[]}}) line 239
2017-11-09 12:21:27.295 [error] <0.671.0>@amqp_gen_connection:terminate:239 Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,"/","localhost",none,[]}) at <0.673.0> exit with reason no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{{function_clause,[{gen,do_for_proc,[{rex,"localhost"},#Fun<gen.0.9..>],...},...]},...}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,"/","localhost",none,[]}}) line 239 in context child_terminated
2017-11-09 12:21:27.295 [error] <0.671.0> Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,"/","localhost",none,[]}) at <0.673.0> exit with reason reached_max_restart_intensity in context shutdown
2017-11-09 12:21:27.295 [error] <0.669.0> CRASH REPORT Process <0.669.0> with 0 neighbours exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{{function_clause,[{gen,do_for_proc,[{rex,"localhost"},#Fun<gen.0.94381411>],[{file,"gen.erl"},{line,253}]},{gen_server,call,3,[{file,"gen_server.erl"},{line,210}]},{rpc,do_call,3,[{file,"rpc.erl"},{line,327}]},{amqp_direct_connection,connect,4,[{file,"src/amqp_direct_connection.erl"},{line,151}]},{amqp_gen_connection,handle_call,3,[{file,"src/amqp_gen_connection.erl"},{line,171}]},{gen_server,try_handle_call,...},...]},...}}}},...},...],...},...]},...} in application_master:init/4 line 134
2017-11-09 12:21:27.295 [info] <0.560.0> Application pampa_pro exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{{function_clause,[{gen,do_for_proc,[{rex,"localhost"},#Fun<gen.0.94381411>],[{file,"gen.erl"},{line,253}]},{gen_server,call,3,[{file,"gen_server.erl"},{line,210}]},{rpc,do_call,3,[{file,"rpc.erl"},{line,327}]},{amqp_direct_connection,connect,4,[{file,"src/amqp_direct_connection.erl"},{line,151}]},{amqp_gen_connection,handle_call,3,[{file,"src/amqp_gen_connection.erl"},{line,171}]},{gen_server,try_handle_call,...},...]},...}}}},...},...],...},...]},...}
2017-11-09 12:25:08.679 [info] <0.560.0> Application lager started on node pampa_pro@localhost
2017-11-09 12:25:08.679 [info] <0.560.0> Application ranch started on node pampa_pro@localhost
2017-11-09 12:25:08.679 [info] <0.560.0> Application ranch_proxy_protocol started on node pampa_pro@localhost
2017-11-09 12:25:08.679 [info] <0.560.0> Application cowboy started on node pampa_pro@localhost
2017-11-09 12:25:08.679 [info] <0.560.0> Application rabbit_common started on node pampa_pro@localhost
2017-11-09 12:25:08.679 [info] <0.560.0> Application amqp_client started on node pampa_pro@localhost
2017-11-09 12:25:08.680 [error] <0.673.0>@amqp_gen_connection:terminate:239 gen_server <0.673.0> terminated with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{{function_clause,[{gen,do_for_proc,[{rex,"localhost"},#Fun<gen.0.9..>],...},...]},...}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,"/","localhost",none,[]}}) line 239
2017-11-09 12:25:08.680 [error] <0.673.0>@amqp_gen_connection:terminate:239 CRASH REPORT Process <0.673.0> with 0 neighbours crashed with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{{function_clause,[{gen,do_for_proc,[{rex,"localhost"},#Fun<gen.0.9..>],...},...]},...}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,"/","localhost",none,[]}}) line 239
2017-11-09 12:25:08.681 [error] <0.671.0>@amqp_gen_connection:terminate:239 Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,"/","localhost",none,[]}) at <0.673.0> exit with reason no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{{function_clause,[{gen,do_for_proc,[{rex,"localhost"},#Fun<gen.0.9..>],...},...]},...}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,"/","localhost",none,[]}}) line 239 in context child_terminated
2017-11-09 12:25:08.681 [error] <0.671.0> Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,"/","localhost",none,[]}) at <0.673.0> exit with reason reached_max_restart_intensity in context shutdown
2017-11-09 12:25:08.681 [error] <0.669.0> CRASH REPORT Process <0.669.0> with 0 neighbours exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{{function_clause,[{gen,do_for_proc,[{rex,"localhost"},#Fun<gen.0.94381411>],[{file,"gen.erl"},{line,253}]},{gen_server,call,3,[{file,"gen_server.erl"},{line,210}]},{rpc,do_call,3,[{file,"rpc.erl"},{line,327}]},{amqp_direct_connection,connect,4,[{file,"src/amqp_direct_connection.erl"},{line,151}]},{amqp_gen_connection,handle_call,3,[{file,"src/amqp_gen_connection.erl"},{line,171}]},{gen_server,try_handle_call,...},...]},...}}}},...},...],...},...]},...} in application_master:init/4 line 134
2017-11-09 12:25:08.681 [info] <0.560.0> Application pampa_pro exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{{function_clause,[{gen,do_for_proc,[{rex,"localhost"},#Fun<gen.0.94381411>],[{file,"gen.erl"},{line,253}]},{gen_server,call,3,[{file,"gen_server.erl"},{line,210}]},{rpc,do_call,3,[{file,"rpc.erl"},{line,327}]},{amqp_direct_connection,connect,4,[{file,"src/amqp_direct_connection.erl"},{line,151}]},{amqp_gen_connection,handle_call,3,[{file,"src/amqp_gen_connection.erl"},{line,171}]},{gen_server,try_handle_call,...},...]},...}}}},...},...],...},...]},...}
2017-11-09 12:25:35.338 [info] <0.560.0> Application lager started on node pampa_pro@localhost
2017-11-09 12:25:35.339 [info] <0.560.0> Application ranch started on node pampa_pro@localhost
2017-11-09 12:25:35.339 [info] <0.560.0> Application ranch_proxy_protocol started on node pampa_pro@localhost
2017-11-09 12:25:35.340 [info] <0.560.0> Application cowboy started on node pampa_pro@localhost
2017-11-09 12:25:35.340 [info] <0.560.0> Application rabbit_common started on node pampa_pro@localhost
2017-11-09 12:25:35.340 [info] <0.560.0> Application amqp_client started on node pampa_pro@localhost
2017-11-09 12:25:35.341 [error] <0.673.0>@amqp_gen_connection:terminate:239 gen_server <0.673.0> terminated with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}}) line 239
2017-11-09 12:25:35.341 [error] <0.673.0>@amqp_gen_connection:terminate:239 CRASH REPORT Process <0.673.0> with 0 neighbours crashed with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}}) line 239
2017-11-09 12:25:35.341 [error] <0.671.0>@amqp_gen_connection:terminate:239 Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}) at <0.673.0> exit with reason no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}}) line 239 in context child_terminated
2017-11-09 12:25:35.341 [error] <0.671.0> Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}) at <0.673.0> exit with reason reached_max_restart_intensity in context shutdown
2017-11-09 12:25:35.342 [error] <0.669.0> CRASH REPORT Process <0.669.0> with 0 neighbours exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@localhost.2.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510201535340},{node,pampa_pro@localhost},{user_who_performed_action,none}]],[]},{rpc,...},...]}}}},...},...],...},...]},...} in application_master:init/4 line 134
2017-11-09 12:25:35.342 [info] <0.560.0> Application pampa_pro exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@localhost.2.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510201535340},{node,pampa_pro@localhost},{user_who_performed_action,none}]],[]},{rpc,...},...]}}}},...},...],...},...]},...}
2017-11-09 12:26:45.861 [info] <0.560.0> Application lager started on node pampa_pro@localhost
2017-11-09 12:26:45.861 [info] <0.560.0> Application ranch started on node pampa_pro@localhost
2017-11-09 12:26:45.861 [info] <0.560.0> Application ranch_proxy_protocol started on node pampa_pro@localhost
2017-11-09 12:26:45.862 [info] <0.560.0> Application cowboy started on node pampa_pro@localhost
2017-11-09 12:26:45.862 [info] <0.560.0> Application rabbit_common started on node pampa_pro@localhost
2017-11-09 12:26:45.862 [info] <0.560.0> Application amqp_client started on node pampa_pro@localhost
2017-11-09 12:26:45.862 [error] <0.673.0>@amqp_gen_connection:terminate:239 gen_server <0.673.0> terminated with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}}) line 239
2017-11-09 12:26:45.862 [error] <0.673.0>@amqp_gen_connection:terminate:239 CRASH REPORT Process <0.673.0> with 0 neighbours crashed with reason: no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}}) line 239
2017-11-09 12:26:45.863 [error] <0.671.0>@amqp_gen_connection:terminate:239 Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}) at <0.673.0> exit with reason no function clause matching amqp_gen_connection:terminate({{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,...],...},...]}}}},...}, {<0.672.0>,{amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}}) line 239 in context child_terminated
2017-11-09 12:26:45.863 [error] <0.671.0> Supervisor {<0.671.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.672.0>, {amqp_params_direct,none,none,<<"/">>,pampa_pro@localhost,none,[]}) at <0.673.0> exit with reason reached_max_restart_intensity in context shutdown
2017-11-09 12:26:45.863 [error] <0.669.0> CRASH REPORT Process <0.669.0> with 0 neighbours exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@localhost.3.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510201605862},{node,pampa_pro@localhost},{user_who_performed_action,none}]],[]},{rpc,...},...]}}}},...},...],...},...]},...} in application_master:init/4 line 134
2017-11-09 12:26:45.863 [info] <0.560.0> Application pampa_pro exited with reason: {{function_clause,[{amqp_gen_connection,terminate,[{{case_clause,{badrpc,{'EXIT',{undef,[{rabbit_direct,connect,[{none,none},<<"/">>,rabbit_framing_amqp_0_9_1,<0.673.0>,[{pid,<0.673.0>},{protocol,{'Direct',{0,9,1}}},{host,unknown},{port,unknown},{name,<<"<pampa_pro@localhost.3.673.0>">>},{peer_host,unknown},{peer_port,unknown},{user,none},{vhost,<<"/">>},{client_properties,[]},{type,direct},{connected_at,1510201605862},{node,pampa_pro@localhost},{user_who_performed_action,none}]],[]},{rpc,...},...]}}}},...},...],...},...]},...}
2017-11-09 12:28:41.920 [info] <0.560.0> Application lager started on node pampa_pro@localhost
2017-11-09 12:28:41.921 [info] <0.560.0> Application ranch started on node pampa_pro@localhost
2017-11-09 12:28:41.921 [info] <0.560.0> Application ranch_proxy_protocol started on node pampa_pro@localhost
2017-11-09 12:28:41.921 [info] <0.560.0> Application cowboy started on node pampa_pro@localhost
2017-11-09 12:28:41.921 [info] <0.560.0> Application rabbit_common started on node pampa_pro@localhost
2017-11-09 12:28:41.921 [info] <0.560.0> Application amqp_client started on node pampa_pro@localhost
2017-11-09 12:28:41.929 [info] <0.560.0> Application pampa_pro started on node pampa_pro@localhost
2017-11-09 12:28:41.932 [info] <0.560.0> Application sasl started on node pampa_pro@localhost
2017-11-09 12:28:41.933 [info] <0.560.0> Application runtime_tools started on node pampa_pro@localhost
2017-11-09 14:02:21.945 [info] <0.560.0> Application lager started on node pampa_pro@localhost
2017-11-09 14:02:21.945 [info] <0.560.0> Application ranch started on node pampa_pro@localhost
2017-11-09 14:02:21.945 [info] <0.560.0> Application ranch_proxy_protocol started on node pampa_pro@localhost
2017-11-09 14:02:21.946 [info] <0.560.0> Application cowboy started on node pampa_pro@localhost
2017-11-09 14:02:21.946 [info] <0.560.0> Application rabbit_common started on node pampa_pro@localhost
2017-11-09 14:02:21.946 [info] <0.560.0> Application amqp_client started on node pampa_pro@localhost
2017-11-09 14:02:21.952 [info] <0.560.0> Application pampa_pro started on node pampa_pro@localhost
2017-11-09 14:02:21.955 [info] <0.560.0> Application sasl started on node pampa_pro@localhost
2017-11-09 14:02:21.956 [info] <0.560.0> Application runtime_tools started on node pampa_pro@localhost
2017-11-09 14:03:41.240 [info] <0.616.0> /home/pampa/pampa_pro/src/pampa_pro_app.erl:0: Recompiled.
2017-11-09 14:03:41.258 [info] <0.616.0> pampa_pro_app: Reloaded! (Beam changed.)
2017-11-09 14:04:12.595 [error] <0.616.0> /home/pampa/pampa_pro/src/ws_handler.erl:1: Error: no module definition
2017-11-09 14:06:03.845 [error] <0.616.0> /home/pampa/pampa_pro/src/msger.erl:1: Error: no module definition
2017-11-09 14:06:16.050 [info] <0.616.0> /home/pampa/pampa_pro/src/msger.erl:0: Recompiled.
2017-11-09 14:08:05.266 [info] <0.616.0> /home/pampa/pampa_pro/src/ws_handler.erl:0: Recompiled.
2017-11-09 14:09:11.043 [info] <0.616.0> /home/pampa/pampa_pro/src/pampa_pro_app.erl:0: Recompiled.
2017-11-09 14:09:11.062 [info] <0.616.0> pampa_pro_app: Reloaded! (Beam changed.)
2017-11-09 14:09:44.452 [info] <0.616.0> /home/pampa/pampa_pro/src/pampa_pro_app.erl:0: Recompiled.
2017-11-09 14:09:44.466 [info] <0.616.0> pampa_pro_app: Reloaded! (Beam changed.)
2017-11-09 14:10:08.778 [info] <0.616.0> /home/pampa/pampa_pro/src/pampa_pro_app.erl:0: Recompiled.
2017-11-09 14:10:08.810 [info] <0.616.0> pampa_pro_app: Reloaded! (Beam changed.)
2017-11-09 14:12:33.390 [info] <0.616.0> /home/pampa/pampa_pro/src/pampa_pro_app.erl:0: Recompiled.
2017-11-09 14:12:33.406 [info] <0.616.0> pampa_pro_app: Reloaded! (Beam changed.)
2017-11-09 14:12:40.510 [info] <0.616.0> /home/pampa/pampa_pro/src/pampa_pro_app.erl:0: Recompiled.
2017-11-09 14:12:40.526 [info] <0.616.0> pampa_pro_app: Reloaded! (Beam changed.)
2017-11-09 14:12:45.611 [info] <0.616.0> /home/pampa/pampa_pro/src/pampa_pro_app.erl:0: Recompiled.
2017-11-09 14:12:45.626 [info] <0.616.0> pampa_pro_app: Reloaded! (Beam changed.)
2017-11-09 14:13:13.973 [info] <0.616.0> /home/pampa/pampa_pro/src/ws_handler.erl:0: Recompiled.
2017-11-09 14:13:13.991 [info] <0.616.0> ws_handler: Reloaded! (Beam changed.)
2017-11-09 14:14:49.026 [info] <0.616.0> /home/pampa/pampa_pro/src/pampa_pro_app.erl:0: Recompiled.
2017-11-09 14:14:49.043 [info] <0.616.0> pampa_pro_app: Reloaded! (Beam changed.)
2017-11-09 14:15:07.239 [info] <0.616.0> /home/pampa/pampa_pro/src/ws_handler.erl:0: Recompiled.
2017-11-09 14:15:07.259 [info] <0.616.0> ws_handler: Reloaded! (Beam changed.)
2017-11-09 14:15:13.369 [info] <0.616.0> /home/pampa/pampa_pro/src/msger.erl:0: Recompiled.
2017-11-09 14:15:13.386 [info] <0.616.0> msger: Reloaded! (Beam changed.)
2017-11-09 14:16:23.169 [info] <0.616.0> /home/pampa/pampa_pro/src/pampa_pro_app.erl:0: Recompiled.
2017-11-09 14:16:23.186 [info] <0.616.0> pampa_pro_app: Reloaded! (Beam changed.)
2017-11-09 14:18:21.470 [info] <0.616.0> /home/pampa/pampa_pro/src/ws_handler.erl:0: Recompiled.
2017-11-09 14:18:21.494 [info] <0.616.0> ws_handler: Reloaded! (Beam changed.)
2017-11-09 14:18:50.793 [info] <0.616.0> /home/pampa/pampa_pro/src/pampa_pro_app.erl:0: Recompiled.
2017-11-09 14:18:50.810 [info] <0.616.0> pampa_pro_app: Reloaded! (Beam changed.)
2017-11-09 14:23:43.955 [error] <0.616.0> /home/pampa/pampa_pro/src/ws_handler.erl:16: Warning: variable 'WsId' is unused
/home/pampa/pampa_pro/src/ws_handler.erl:19: Error: record ws_state undefined
/home/pampa/pampa_pro/src/ws_handler.erl:40: Warning: variable 'MsgerPid' is unused
/home/pampa/pampa_pro/src/ws_handler.erl:42: Error: record ws_state undefined
/home/pampa/pampa_pro/src/ws_handler.erl:44: Error: record ws_state undefined
/home/pampa/pampa_pro/src/ws_handler.erl:59: Error: record ws_state undefined
/home/pampa/pampa_pro/src/ws_handler.erl:65: Error: record ws_state undefined
/home/pampa/pampa_pro/src/ws_handler.erl:90: Error: record ws_state undefined
/home/pampa/pampa_pro/src/ws_handler.erl:91: Error: record ws_state undefined
2017-11-09 14:25:56.451 [error] <0.616.0> /home/pampa/pampa_pro/src/msger.erl:9: Warning: variable 'WsPid' is unused
/home/pampa/pampa_pro/src/msger.erl:26: Warning: variable 'Tag' is unused
/home/pampa/pampa_pro/src/msger.erl:36: Error: record msger_state undefined
/home/pampa/pampa_pro/src/msger.erl:58: Error: record msger_state undefined
/home/pampa/pampa_pro/src/msger.erl:63: Error: record msger_state undefined
/home/pampa/pampa_pro/src/msger.erl:68: Error: record msger_state undefined
2017-11-09 14:26:05.573 [error] <0.616.0> /home/pampa/pampa_pro/src/msger.erl:9: Warning: variable 'WsPid' is unused
/home/pampa/pampa_pro/src/msger.erl:26: Warning: variable 'Tag' is unused
/home/pampa/pampa_pro/src/msger.erl:36: Error: record msger_state undefined
/home/pampa/pampa_pro/src/msger.erl:58: Error: record msger_state undefined
/home/pampa/pampa_pro/src/msger.erl:63: Error: record msger_state undefined
/home/pampa/pampa_pro/src/msger.erl:68: Error: record msger_state undefined
2017-11-09 14:26:19.757 [error] <0.616.0> /home/pampa/pampa_pro/src/msger.erl:9: Warning: variable 'WsPid' is unused
/home/pampa/pampa_pro/src/msger.erl:26: Warning: variable 'Tag' is unused
/home/pampa/pampa_pro/src/msger.erl:36: Error: record msger_state undefined
/home/pampa/pampa_pro/src/msger.erl:58: Error: record msger_state undefined
/home/pampa/pampa_pro/src/msger.erl:63: Error: record msger_state undefined
/home/pampa/pampa_pro/src/msger.erl:68: Error: record msger_state undefined
2017-11-09 14:52:19.602 [info] <0.562.0> Application lager started on node pampa_pro@localhost
2017-11-09 14:52:19.602 [info] <0.562.0> Application ranch started on node pampa_pro@localhost
2017-11-09 14:52:19.602 [info] <0.562.0> Application ranch_proxy_protocol started on node pampa_pro@localhost
2017-11-09 14:52:19.602 [info] <0.562.0> Application cowboy started on node pampa_pro@localhost
2017-11-09 14:52:19.602 [info] <0.562.0> Application rabbit_common started on node pampa_pro@localhost
2017-11-09 14:52:19.603 [info] <0.562.0> Application amqp_client started on node pampa_pro@localhost
2017-11-09 14:52:19.603 [info] <0.562.0> Application pampa_pro started on node pampa_pro@localhost
2017-11-09 14:52:19.605 [info] <0.562.0> Application sasl started on node pampa_pro@localhost
2017-11-09 14:52:19.605 [info] <0.562.0> Application runtime_tools started on node pampa_pro@localhost
2017-11-09 14:57:35.308 [info] <0.583.0> SIGTERM received - shutting down
2017-11-09 15:01:44.754 [info] <0.562.0> Application lager started on node pampa_pro@localhost
2017-11-09 15:01:44.754 [info] <0.562.0> Application ranch started on node pampa_pro@localhost
2017-11-09 15:01:44.754 [info] <0.562.0> Application ranch_proxy_protocol started on node pampa_pro@localhost
2017-11-09 15:01:44.754 [info] <0.562.0> Application cowboy started on node pampa_pro@localhost
2017-11-09 15:01:44.754 [info] <0.562.0> Application rabbit_common started on node pampa_pro@localhost
2017-11-09 15:01:44.754 [info] <0.562.0> Application amqp_client started on node pampa_pro@localhost
2017-11-09 15:01:44.755 [info] <0.562.0> Application pampa_pro started on node pampa_pro@localhost
2017-11-09 15:01:44.758 [info] <0.562.0> Application sasl started on node pampa_pro@localhost
2017-11-09 15:01:44.758 [info] <0.562.0> Application runtime_tools started on node pampa_pro@localhost
